packages/garmin-bridge/test/chrome-mock.js

```
@@ -16,13 +16,21 @@ const chromeMock = {
     onMessageExternal: {
       addListener: vi.fn(),
     },
+    onInstalled: {
+      addListener: vi.fn(),
```

ESLint

[error] 'vi' is not defined.

(no-undef)